src/FiscalYears/EditFiscalYear/EditFiscalYear.js

```diff
@@ -40,7 +41,11 @@ export const EditFiscalYear = ({ resources, mutator, match, history, location })
     [fiscalYearId, location.search],
   );
 
-  const saveFiscalYear = useSaveFiscalYear(mutator.fiscalYearEdit, closeEdit, 'PUT');
+  const { saveFiscalYear, error } = useSaveFiscalYear(mutator.fiscalYearEdit, closeEdit, 'PUT');
+
+  useEffect(() => {
```

Comment:
Do you really need this? Could you use just `error` from `useSaveFiscalYear`?

Reply:
Yeah) why not, thanks